src/com/example/ui.cljs

```diff
@@ -91,7 +93,26 @@
            {:header (comp/get-query Header)}
            {:quiz (comp/get-query Quiz)}]
    :initial-state {:root/user {} :header {} :quiz {}}}
-  (div :.container.mx-auto.text-gray-100
+  (div :.text-gray-100
        (ui-header (comp/computed (:header props) {:on-signout #(comp/transact! this [(mut/delete-root-user nil)]) :user user}))
-       (when (:quiz props)
-         (ui-quiz (:quiz props)))))
+       (when (not (:user/authenticated? user))
+         (div :.grid.md:grid-cols-2.grid-cols-1
+              (div :.md:grid-span-1.md:min-h-screen.w-full.py-8.bg-gray-600.rounded.mb-16
+                   (div :.flex.flex-col.place-content-center.h-full.mb-8
```

Comment:
first time using `place-content-center` but was really easy way to center a div in the middle of the screen

Reply:
oh i had no idea about `place-content`